processor/package.json

```diff
@@ -2,23 +2,27 @@
   "name": "kitspace-processor",
   "version": "0.0.1",
   "description": "Asset processor for kitspace-v2",
-  "main": "index.js",
   "license": "AGPL-3.0",
   "private": true,
+  "type": "module",
+  "exports": "./index.js",
+  "engines": {
+    "node": ">=16"
+  },
   "scripts": {
-    "dev": "nodemon src/server.ts --watch src/",
-    "start": "ts-node src/server.ts",
-    "test": "LOG_LEVEL=debug DATA_DIR=/data/test mocha test/integration/**/*.ts -r ts-node/register --timeout=120000 --exit",
-    "posttest": "rm -rf /data/test",
-    "tsc": "tsc",
+    "dev": "tsc-watch --noClear --onSuccess 'bash -c \"yarn cp-assets && node ./dist/src/server.js\"'",
+    "cp-assets": "find ./src ! -name '*.ts' -exec cp -r '{}' ./dist/src/ \\;",
+    "cp-test-assets": "find ./test ! -name '*.ts' -exec cp -r '{}' ./dist/test/ \\;",
+    "test": "mocha dist/test/integration/**/*.js --timeout=120000 --exit",
```

Comment:
I really wanted to stick to running the integration tests on the .ts files. Now we will have to mentally map the .js source file location to the .ts file location when we get a stack trace.

Reply:
A thought I had when I was working on this is to seperate out the integration tests into it's own container. Then run them either on the dev or production server as needed.

Reply:
In 9552d89, I added support for [source maps](https://www.typescriptlang.org/tsconfig#sourceMap).

|before|after|
|---|---|
|![image](https://user-images.githubusercontent.com/37152329/189239366-32a3d0df-7f16-4562-90f7-c5c5ef3c6e7d.png)|![image](https://user-images.githubusercontent.com/37152329/189239014-872c2b36-983c-446d-97b0-6765cde144c8.png)|